src/ert/scheduler/job.py

```diff
@@ -73,7 +73,7 @@ async def _submit_and_run_once(self, sem: asyncio.BoundedSemaphore) -> None:
                 self.real.iens, self.real.job_script, cwd=self.real.run_arg.runpath
             )
 
-            await self._send(State.STARTING)
+            await self._send(State.STARTING)  # aka PENDING
```

Comment:
Is `PENDING` a better name? Should we change it back so that we don't need this comment?

Reply:
Perhaps not better by itself, but more familiar for legacy ert devs..

Reply:
Will change back to PENDING. I will claim it means exactly the same, and also makes as much sense as STARTING for the LocalDriver.